<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_scientist</class>
 <widget class="QDialog" name="Add_scientist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>First name(s):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_first"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Last name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_last"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Date of birth:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="birth_edit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date of death:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="death_edit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gender_select">
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="living">
       <property name="text">
        <string>Living</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="add_button">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
